alex/applications/PublicTransportInfoCS/data/ontology.py

```diff
@@ -302,7 +302,8 @@
         'lta_task': {
             'weather': [('', '^task$', '^weather$'), ],
             'find_connection': [('', '^task$', '^find_connection$'), ('', '^departure_', ''), ('', '^arrival_', ''),
-                                ('', '^duration$', '')],
+                                ('', '^duration$', ''), 
+                                ('', '^from_stop$', ''), ('', '^to_stop$', ''),],
```

Comment:
This will not play well with the `find_platform` task below, as the last-talked-about inference will switch to the `find_connection` task whenever the user mentions `from_stop` or `to_stop`.